docs/content/troubleshooting.md

```diff
@@ -1,113 +1,121 @@
-[//]: # (Doc of docs reference 23)
 [//]: # (TODO - tech writer pass)
 
 # Troubleshooting
 
-* [Errors](#errors)
-* [Exceptions](#exceptions)
-* [Problems](#problems)
-
-We also maintain a [known issues]({{urlRoot}}/known-issues) page for any open bugs in the SpatialOS GDK for Unity.
+<%(TOC)%>
 
+<%(Callout type="tip" message="We also maintain a [known issues]({{urlRoot}}/known-issues) page for any open bugs in the SpatialOS GDK for Unity.")%>
 
 ## Errors
 
-#### Error reported
-When building workers (Unity Editor menu: **Build For Cloud** > **my_worker_name**):
+#### Error building player because build target was unsupported
+
+When building workers by selecting **Build For Cloud** > **my_worker_name** the following error may occur:
 **_Error building player because build target was unsupported_**</br>
 
-**Issue and workarounds**<br/>
- You don't have the right Unity build settings enabled.<br/>
- In your Unity Editor:
+**Cause**<br/>
+ You don't have the correct Unity build support packages installed.<br/>
+
+**Fix**<br/>
+ In the Unity Editor:
 
-* Check what build support you have enabled (menu: **File** > **Build Settings**) and make sure you have your target build platform enabled. <br/>
+* Select **SpatialOS** > **Check build support** and check the Console for errors relating to specific build platforms. <br/>
   * You need **Mac** build support if you are developing on a Windows PC and want to share your game with Mac users.
-  * You need **Windows** build support if you are developing on a Mac and want to share your game with Windows PC users. 
+  * You need **Windows** build support if you are developing on a Mac and want to share your game with Windows PC users.
+  * You need **Android** and/or **iOS** build support if you are developing for those platforms.
   * Unity gives you build support for your development machine (Windows or Mac) by default.
-  * _**In addition**_, make sure you have **Linux** build support enabled.<br/> 
- You need Linux build support because all server-workers in a cloud deployment run in a Linux environment. 
- <br/>Fix this by runing the Unity installer and selecting the appropriate option. (See [Setup and installing]({{urlRoot}}/setup-and-installing#set-up-your-machine)).
+  * _**In addition**_, make sure you have **Linux** build support enabled. You need Linux build support because all server-workers in a cloud deployment run in a Linux environment.
+
+ Fix this by runing the Unity installer and selecting the appropriate build support options during the instalation. See [Setup and installing]({{urlRoot}}/setup-and-installing#set-up-your-machine) for more information.
```

Comment:
`instalation` vs `installation`. I think we prefer the American spelling

Reply:
fixed